packages/devtools_app/lib/src/performance/timeline_flame_chart.dart

```diff
@@ -1012,6 +1189,224 @@ class SelectedFrameBracketPainter extends FlameChartPainter {
       );
 }
 
+class PreviousInThreadButton extends StatefulWidget {
```

Comment:
PreviousInThreadButton and NextInThreadButton are basically the same. Merge into one class.